e2e/testcases/helm_sync_test.go

```diff
@@ -98,6 +100,13 @@ func TestPublicHelm(t *testing.T) {
 
 	nt.T.Log("Update RootSync to sync from a public Helm Chart without specified release namespace")
 	nt.MustMergePatch(rs, `{"spec": {"helm": {"namespace": ""}}}`)
+	nt.T.Cleanup(func() {
```

Comment:
Is this `Cleanup` associated with the change in namespace on the line above? From reading this it sounds like it's associated with the initial apply on line `61`. In which case, I'm worried we won't reach this `Cleanup` block before the validations fail. Am I missing something? If not, can this `Cleanup` be moved up to around line `62` to ensure it runs?